server/api/settings.py

```diff
@@ -12,6 +12,12 @@
 import os
 from pathlib import Path
 
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "root": {"level": "INFO"}
+}
+
```

Comment:
Is this needed? Where does the logging occur?

Reply:
oh that was just for debugging the server code